pyproject.toml

```diff
@@ -51,7 +51,7 @@ dev = [
     "numpydoc",
 
     # For schema generation.
-    "pydantic<2.0",
+    "pydantic~=2.6.0",
```

Comment:
In library code, as opposed to application code, we generally avoid pinning things _down_, as this can create more problems than it solves. (Gallons of proverbial blog post ink have been spilled on this subject: [here's one](https://iscinumpy.dev/post/bound-version-constraints/) that links out to others if anyone wants to fall down the rabbithole.)

Reply:
Good post! I'll pin >= 2.6